CMakeLists.txt

```diff
@@ -270,3 +269,18 @@ target_sources(cquery PRIVATE
                src/messages/workspace_did_change_configuration.cc
                src/messages/workspace_did_change_watched_files.cc
                src/messages/workspace_symbol.cc)
+
+### Tools
+
+# We use glob here since source files are already manually added with
+# target_sources
+file(GLOB_RECURSE SOURCES src/*.cc src/*.h)
+
+if(Clang_FORMAT)
+  add_custom_command(TARGET cquery
```

Comment:
I think we should only do this if using bundled clang, since different clang-format versions give different output.